src/create_disk.sh

```diff
@@ -30,14 +31,63 @@ Options:
     --help: show this help
     --kargs: kernel CLI args
     --no-x86-bios-bootloader: don't install BIOS bootloader on x86_64
+    --with-secure-execution:  enable IBM SecureExecution
 
 You probably don't want to run this script by hand. This script is
 run as part of 'coreos-assembler build'.
 EOC
 }
 
+# IBM SecureExecution variables
+secure_execution_hostkey="/dev/disk/by-id/virtio-hostkey"
+secure_execution_initrd="/tmp/initrd.secure"
+secure_execution_bootfs_key="boot"
+secure_execution_bootfs_label="crypt_bootfs"
+secure_execution_bootfs_device="/dev/mapper/crypt_bootfs"
+secure_execution_rootfs_key="root"
+secure_execution_rootfs_label="crypt_rootfs"
+secure_execution_rootfs_device="/dev/mapper/crypt_rootfs"
```

Comment:
My suggestion is to drop these variables and just pass `boot` or `root` as needed to `create_luks_partition` and `propagate_luks_config`. All the values can be derived from those (e.g. `/dev/mapper/crypt_${label}fs`).